core/messaging/src/providers/whisper/index.ts

```diff
@@ -0,0 +1,7 @@
+import { getWeb3 } from './web3Utils';
+import { WhisperService } from './whisper';
+
+export {
```

Comment:
Has Whisper been left in for backwards compatibility?

Reply:
It was left in so we could show the simplicity of adding new implementations as well as giving us the option to fully migrate radish34 to use the new packages. In fact, @bitwiseguy did this migration insofar as the radish34 messenger package is concerned.